Libraries/src/Amazon.Lambda.Tools/Program.cs

```diff
@@ -12,6 +12,23 @@ namespace Amazon.Lambda.Tools
 {
     public class Program
     {
+        private static string Version
+        {
+            get
+            {
+                AssemblyInformationalVersionAttribute attribute = null;
+                try
+                {
+                    attribute = Assembly.GetEntryAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>();
+                }
+                catch (AmbiguousMatchException e)
+                {
+                    Console.Error.WriteLine(e.Message);
```

Comment:
you may want to just return null in that case instead of printing a nasty exception